pagopa/io-app
# chore: [IOCOM-1761] Push notification reminder banner BS and logic

## Short description
this PR adds the complete logic for the push notification reminder banner in the homescreen, along with its bottomSheet.
<img src="https://github.com/user-attachments/assets/5f27f74c-125a-48b2-92d7-3e32b27e63b7" width=250 />

## List of changes proposed in this pull request
- required I18n
- required persisted data
- new store migration
- addition of bottomSheet UI and logic
- required selectors
- required actions

## How to test
with notifications disabled, log in the app using biometric/pin, and make sure that, as long as you haven't seen the push notifications reminder alert, the banner renders correctly.

then, while using the dev-server, close the banner 3 times ( you should need to close and re-open the app each time for the banner to show up again ), the third time you should be prompted with a bottomSheet.

in this stage, check that:

1. tapping on "remind me later" removes the banner like the previous times, and it shows up on next app open
2. closing works as expected (both x and tapping out)
3. tapping on "never show again" hides the banner, and also does in the next launches
4. reloading the dev server (then feeding the app at least 4 new unread messages), shows the banner again, and you can now repeat the full procedure listed earlier.
